ui-components/src/locales/general.json

```diff
@@ -1257,6 +1257,21 @@
       "prompt" : "Do you have a disability?",
       "description" : "Some properties are meant for residents with disabilities. Some homes might request proof of disability with your application.",
       "preferNotToSay": "Prefer not to say"
+    },
+    "income" : {
```

Comment:
Do we need translations for these "income" fields for other languages? (In, e.g. ui-components/src/locales/es.json?)

Reply:
Yes, we would need to translate them, but we don't have any of our required translations set up yet.

Reply:
One thing I've done is to just plug them in to Google Translate, and then copy-paste the output to those other JSON files. Without that, I think the way this currently works is that if the user clicks e.g. "Espanol", it'll stay in English, right? So I might favor a Google-Translate Spanish translation over just keeping the page in English, but I'll leave the decision to you! (i.e. feel free to just resolve this thread :)

Reply:
Good point! I'll actually do that in a separate PR because I have to translate all of the other eligibility strings and add the other languages we need.